packages/block-library/src/pullquote/theme.scss

```diff
@@ -1,13 +1,13 @@
 .wp-block-pullquote {
-	border-top: 4px solid $dark-gray-500;
-	border-bottom: 4px solid $dark-gray-500;
+	border-top: 4px solid #555;
```

Comment:
Could these be `currentColor` since you define the color?

Reply:
That's a great thought, and I was going to implement it. However that makes this a bigger change — 555 is basically just a neutral gray version of the same color that was there before, which is very middle of the road gray. `currentColor` is going to be the same color as the text — which I agree it should be, because look:

```
.wp-block-pullquote {
	border-top: 4px solid #555;
	border-bottom: 4px solid #555;
	margin-bottom: 1.75em;
	color: #555;

	cite,
	footer,
	&__citation {
		color: #555;
		text-transform: uppercase;
		font-size: 0.8125em;
		font-style: normal;
	}
}
````

The body text is also gray. So it's not super arbitrary-background friendly. 

But it also makes it a bigger change so I'm wondering if it shouldn't be a separate PR?

Reply:
Separate PR works for me.

Reply:
Followup here: https://github.com/WordPress/gutenberg/pull/25358